src/apps/opening-hours-editor/OpeningHoursEditor.entry.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { withText } from "../../core/utils/text";
+import { withUrls } from "../../core/utils/url";
+import OpeningHoursEditor, {
+  OpeningHoursEditorType
+} from "./OpeningHoursEditor";
+import { withConfig } from "../../core/utils/config";
+
+interface OpeningHoursEditorEntryTextProps {
+  openingHoursRemoveEventButtonText: string;
+}
+
+interface OpeningHoursEditorEntryConfigProps {
+  openingHoursEditorCategoriesConfig: string;
+  openingHoursBranchIdConfig: string;
+}
+
+const OpeningHoursEditorEntry: React.FC<
+  OpeningHoursEditorEntryTextProps &
+    OpeningHoursEditorType &
+    OpeningHoursEditorEntryConfigProps
+> = ({ useWireMockStartDate }) => (
+  <OpeningHoursEditor useWireMockStartDate={useWireMockStartDate} />
```

Comment:
Please use the start date as a prop instead of a boolean referencing Wiremock.

In general we do not want Wiremock specific code inside our components. It should be controlled using Storybook controls or the like as it is used for development. Then in the Storybook we can set a default for the start date prop and add a comment referencing Wiremock there.